app/controllers/summary_controller.rb

```diff
@@ -2,27 +2,38 @@ class SummaryController < ApplicationController
   before_action :authorize
 
   def index
-    client_namespace = current_user.client_slug.titleize
-    @client_summarizer = ClientSummarizer.new(
-      client_namespace: client_namespace,
-      fiscal_year: params[:fiscal_year]
-    )
-    @summary = @client_summarizer.run
+    client_namespaces = if current_user.gateway_admin?
```

Comment:
1 thoughts:

1. should regular admins also be able to see all proposals in the summary view? 

2. It seems like this logic should go into its own method -- perhaps a helper method we can test?